src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,45 @@
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+export const Navigation = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            className={({ isActive }) => (
+              cn(
+                'navbar-item',
+                {
+                  'has-background-grey-lighter': isActive,
+                },
+              )
+            )}
+            to="/"
+          >
```

Comment:
lets save up some space
```suggestion
          <NavLink
            to="/"
            className={({ isActive }) => (
              cn('navbar-item', {
                'has-background-grey-lighter': isActive,
              }))}
          >
```